packages/eslint-plugin/docs/rules/README.md

```diff
@@ -6,18 +6,24 @@ pagination_prev: null
 
-## Supported Rules
+Instead of enabling rules one by one, we recommend using one of [our pre-defined configs](/linting/configs) to easily enable a large set of recommended rules.
 
-import RulesTable from "@site/src/components/RulesTable";
+## Rules
+
+The rules are listed in alphabetical order. You can optionally filter them based on these categories:
 
-<RulesTable ruleset="supported-rules" />
+import RulesTable from "@site/src/components/RulesTable";
 
-## Extension Rules
+<RulesTable />
 
-In some cases, ESLint provides a rule itself, but it doesn't support TypeScript syntax; either it crashes, or it ignores the syntax, or it falsely reports against it.
-In these cases, we create what we call an extension rule; a rule within our plugin that has the same functionality, but also supports TypeScript.
+## Filtering
 
-<RulesTable ruleset="extension-rules" />
+- "Config Group" refers to the configuration preset. We offer [three different config groups](/linting/configs) that allow users to enable a large set of recommended rules all at once.
+- `🔧 fixable` refers to whether the rule contains an ESLint auto-fixer. If the rule has an auto-fixer, then some rule violations can be fixed by running `eslint` with the `--fix` flag. This will automatically change the code, which can save a lot of time! (It is a common pattern for developers to configure their IDE to automatically run `eslint --fix` when saving a TypeScript file.)
+- `💡 has suggestions` refers to whether the rule will offer a suggestion of how to fix the code. Sometimes, it is not safe to automatically fix the code with an auto-fixer. But in these cases, we often have a good guess of what the correct fix should be, and we can provide it as a suggestion to the developer.
+- `💭 requires type information` refers to whether the rule needs to leverage the power of the TypeScript compiler in order to work properly. Rules that require type information are usually much slower than rules that don't. Thus, if linting performance becomes an issue in a gigantic codebase, a good first step might be make rules that require type information run only in CI.
+- `🧱 extension rule` means that the rule was originally a [core ESLint rule](https://eslint.org/docs/latest/rules/). Some core ESLint rules do not support TypeScript syntax: either they crash, ignore the syntax, or falsely report against it. In these cases, we create an "extension rule": a rule within our plugin that has the same functionality, but also supports TypeScript.
+- `📐 formatting rule` means that the rule has to do with formatting. Formatting rules are mostly here for legacy purposes, because we [strongly recommend against using ESLint for formatting](/linting/troubleshooting/formatting).
+- `💀 deprecated rule` means that the rule should no longer be used and will be removed from the plugin in a future version.
```

Comment:
[Content] Moving this all down to the bottom of the page does help make the page easier to skim. But it doesn't address the issue that this text is quite a lot to read. Which I think is a symptom of a deeper issue: that the text is duplicating docs that should really exist elsewhere (and mostly do). We don't need to re-explain, say, what fixers are, or what typed linting is. There are other docs for that - and we want people to all read the same docs.

What do you think about splitting this into smaller list items, each with links to relevant docs?

```md
### Config Group (⚙️)

"Config Group" refers to the [pre-defined config](/linting/configs) that includes the rule.
Extending from a configuration preset allow for enabling a large set of recommended rules all at once.

### Metadata

- `🔧 fixable` refers to whether the rule contains an [ESLint `--fix` auto-fixer](https://eslint.org/docs/latest/use/command-line-interface#--fix).
- `💡 has suggestions` refers to whether the rule contains an ESLint suggestion fixer.
  - Sometimes, it is not safe to automatically fix the code with an auto-fixer. But in these cases, we often have a good guess of what the correct fix should be, and we can provide it as a suggestion to the developer.
- `💭 requires type information` refers to whether the rule requires [typed linting](/linting/typed-linting).
- `🧱 extension rule` means that the rule was originally a [core ESLint rule](https://eslint.org/docs/latest/rules/).
  - Some core ESLint rules do not support TypeScript syntax: either they crash, ignore the syntax, or falsely report against it.
  - In these cases, we create an "extension rule": a rule within our plugin that has the same functionality, but also supports TypeScript.
- `📐 formatting rule` means that the rule has to do with formatting.
  - We [strongly recommend against using ESLint for formatting](/linting/troubleshooting/formatting).
  - Soon, formatting rules will be moved to the [ESLint stylistic plugin](https://eslint.style).
- `💀 deprecated rule` means that the rule should no longer be used and will be removed from the plugin in a future version.
```

The only points here that aren't covered elsewhere are:

* Suggestion fixers: which should really be documented on eslint.org. I'll file an issue.
* Extension rules: good info, left in - but split into sub-items for readability
    * Now I wonder: should we have more dedicated docs about them? 🤔 
* Formatting rules: eslint.style has become the right place to point folks to 🚀 

I don't want to be a stick in the mud and keep dragging on this PR. I'd be up for merging if you two are happy with how things are now. But I do think we can trim this down a bit. Thoughts?

cc @Josh-Cena